share/commands/agent-health

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+# @description monitor agent health
+# @man Check that rudder agent has no problem
+# @man +
+# @man *Options*:
+# @man +
+# @man *-n*: run in nrpe mode, print a single line and return 0,1 or 2
+# @man  put this line in your nrpe.cfg to use it
+# @man  command[check_rudder]=/opt/rudder/bin/rudder agent health -n
+
+CFE_DIR=/var/rudder/cfengine-community
+ENTERPRISE_CFE_DIR=/var/cfengine
+CFE_DISABLE_FILE=/opt/rudder/etc/disable-agent
+UUID_FILE=/opt/rudder/etc/uuid.hive
+
+[ -e "${ENTERPRISE_CFE_DIR}/bin/cf-agent" ] && CFE_DIR="${ENTERPRISE_CFE_DIR}"
+
+while getopts "n" opt; do
+  case $opt in
+    n)
+      # ignored for the moment but provided for forward compatibility
+      NRPE=y
+      ;;
+  esac
+done
+
+# test if disabled
+if [ -e "${CFE_DISABLE_FILE}" ]
+then
+  echo "Rudder agent disabled"
+  exit 1 # warning
+fi
+
+# test policy server
+if [ ! -e "${CFE_DIR}/policy_server.dat" ] || grep "%" "${CFE_DIR}/policy_server.dat" > /dev/null
+then
+  echo "Rudder agent not configured"
+  exit 2 # error
+fi
+
+# test policy server value
+cat "${CFE_DIR}/policy_server.dat" | xargs getent hosts > /dev/null
+if [ $? -ne 0 ]
+then
+  echo "Uknown Rudder policy server"
+  exit 2 # error
+fi
+
+# test connection errors
+egrep "FATAL:|Fatal :|Could not get an updated configuration" "${CFE_DIR}/output/previous" > /dev/null
+if [ $? -ne 0 ]
+then
+  echo "Connection errors in Rudder agent last run"
```

Comment:
Yes, it does:

    /var/rudder/configuration-repository/techniques/system/common/1.0/update.st:133:* rudder-agent could not get an updated configuration from the policy server.   *

However, that should not be a capital C, therefore